src/components/MenuItem.tsx

```diff
@@ -469,6 +469,7 @@ function MenuItem(
             interactive && disabled ? {...styles.userSelectNone} : {},
             styles.ltr,
             isDeleted ? styles.offlineFeedback.deleted : {},
+            styles.breakWord,
```

Comment:
I think this needs to be an optional style since `MenuItem` is used across the app in many different ways.